joomla/joomla-cms
# Some missing jquery framework dependency

### Description

This PR adds some missing jquery framework dependency.
### How to test
- In admin go to Content -> Media and check there is no javascript errors in the js console.
- In admin go to System -> Global Check in and check there is no javascript errors in the js console.

You can also check the code to see jquery js are loaded.
### B/C

None
